### PR TITLE
Update QD to QL 1

### PR DESCRIPTION
**Pull request by brawner**
@chapulina tagging you here because this is in the ros-tooling org.
Signed-off-by: Stephen Brawner <brawner@gmail.com>

### CODE REVIEW

File: QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -145,31 +153,31 @@ Currently nightly test results can be seen here:
 
 The `rcl` package supports implementation of language specific ROS Client Libraries.
 
-It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcl/blob/master/rcl/QUALITY_DECLARATION.md).
+It is **Quality Level 1**, see its [Quality Declaration document](https://github.com/ros2/rcl/blob/master/rcl/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:156]: Depends on https://github.com/ros2/rcl/pull/866

File: QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -145,31 +153,31 @@ Currently nightly test results can be seen here:
...
 
-It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcpputils/blob/master/QUALITY_DECLARATION.md).
+It is **Quality Level 1**, see its [Quality Declaration document](https://github.com/ros2/rcpputils/blob/master/QUALITY_DECLARATION.md).
 
 #### `rosidl_default_runtime`
 
 The `rosidl_default_runtime` provides CMake functionality for finding and adding runtime dependencies for rosidl packages.
 
-It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rosidl_defaults/blob/master/rosidl_default_runtime/QUALITY_DECLARATION.md).
+It is **Quality Level 1**, see its [Quality Declaration document](https://github.com/ros2/rosidl_defaults/blob/master/rosidl_default_runtime/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:168]: Depends on https://github.com/ros2/rosidl_defaults/pull/15

File: QUALITY_DECLARATION.md [hunk 3]
```diff
@@ -145,31 +153,31 @@ Currently nightly test results can be seen here:
...
 
 #### `statistics_msgs`
 
 The `statistics_msgs` package contains ROS 2 message definitions for reporting statistics for topics and system resources.
 
-It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/rcl_interfaces/blob/master/statistics_msgs/QUALITY_DECLARATION.md).
+It is **Quality Level 1**, see its [Quality Declaration document](https://github.com/ros2/rcl_interfaces/blob/master/statistics_msgs/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:174]: Depends on https://github.com/ros2/rcl_interfaces/pull/116

File: QUALITY_DECLARATION.md [hunk 4]
```diff
@@ -145,31 +153,31 @@ Currently nightly test results can be seen here:
...
 
 #### `std_msgs`
 
 The `std_msgs` package provides many basic message types.
 
-It is **Quality Level 2**, see its [Quality Declaration document](https://github.com/ros2/common_interfaces/blob/master/std_msgs/QUALITY_DECLARATION.md).
+It is **Quality Level 1**, see its [Quality Declaration document](https://github.com/ros2/common_interfaces/blob/master/std_msgs/QUALITY_DECLARATION.md).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:180]: Depends on https://github.com/ros2/common_interfaces/pull/135

**Review by chapulina — APPROVED** [review]
LGTM after the upstream PRs are merged.

**Comment by chapulina** [comment]
All upstream PRs are in. Could some maintainer take a quick look and get this in? Maybe @dabonnie ? Thanks!

**Review by prajakta-gokhale — APPROVED** [review]

**Review by dabonnie — APPROVED** [review]

**Comment by chapulina** [comment]
Looks like we're close to merging! :smirk:  :innocent:

**Pull request merged**